### PR TITLE
added alias for react-router, to solve double-router bug

### PR DESCRIPTION
**Pull request by spacerockzero**
^^
Needed for zion-analytics, which has a `<Link/>` HOC, to prevent it from throwing "Invariant" errors while NPM linked, due to it creating 2 nested router scopes

### CODE REVIEW

**Comment by spacerockzero** [comment]
has a dep issue currently

**Comment by spacerockzero** [comment]
should this dep be zion-router, or react-router?

File: packages/react-scripts/config/webpack.config.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ const sassModuleRegex = /\.module\.(scss|sass)$/;
 
 // This is the production and development configuration.
 // It is focused on developer experience, fast rebuilds, and a minimal bundle.
-module.exports = function(webpackEnv) {
+module.exports = function (webpackEnv) {
```
Comment (joeycozza) [inline packages/react-scripts/config/webpack.config.js]: We try our best not to edit this file with our prettier config, that way in the future it makes merging upstream changes easier on us. Could you undo the whitespace/prettier changes for this file?

Comment (spacerockzero) [reply packages/react-scripts/config/webpack.config.js]: sure thing. been only working inside zion code lately, so i forgot to turn it off, it was on auto

**Comment by spacerockzero** [comment]
closing for fixed: https://github.com/fs-webdev/create-react-app/pull/170

**Pull request closed without merging**